src/main/java/com/kainos/ea/models/Role.java

```diff
@@ -0,0 +1,101 @@
+package com.kainos.ea.models;
+
+import com.kainos.ea.enums.Locations;
+import com.kainos.ea.enums.RoleStatus;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+public class Role {
```

Comment:
I would name this file RoleResponse as Role suggests that this is what the Role table looks like

Reply:
done